staging_vespalib/src/vespa/vespalib/util/process_memory_stats.h

```diff
@@ -37,6 +36,7 @@ class ProcessMemoryStats
     uint64_t getMappingsCount() const { return _mappings_count; }
     bool similarTo(const ProcessMemoryStats &rhs, uint64_t sizeEpsilon) const;
     vespalib::string toString() const;
+    bool operator < (const ProcessMemoryStats & rhs) const { return _mapped_rss < rhs._mapped_rss; }
```

Comment:
We are using _anonymous_rss when calculating memory used ratio, so I guess we should use that when ordering as well:
`
double
DiskMemUsageFilter::getMemoryUsedRatio(const Guard &guard) const
{
    (void) guard;
    uint64_t unscaledMemoryUsed = _memoryStats.getAnonymousRss();
    return static_cast<double>(unscaledMemoryUsed) / _hwInfo.memory().sizeBytes();
}
`

Reply:
Ok, I assumed that mapped was both file backed and anonymous. @toregge what say you.